Tests/test_SeqIO_index.py

```diff
@@ -26,19 +26,14 @@
 from Bio.SeqIO._index import _FormatToRandomAccess
 from Bio.Alphabet import generic_protein, generic_nucleotide, generic_dna
 
-from seq_tests_common import compare_record
-
-from Bio import StreamModeError
-
 from Bio import BiopythonParserWarning
 from Bio import MissingPythonDependencyError
 
-CUR_DIR = os.getcwd()
+from seq_tests_common import compare_record
+from test_SeqIO import SeqIOTestsBaseClass
 
 
-def add_prefix(key):
```

Comment:
Moving ``add_prefix`` from a function to a method is fine, although the comments are a little misleading after doing so. From testing the API either way is fine.